src/openrct2/ride/RideData.h

```diff
@@ -348,7 +362,8 @@ constexpr const RideTypeDescriptor DummyRTD =
     SET_FIELD(BonusValue, 0),
     SET_FIELD(ColourPresets, DEFAULT_FLAT_RIDE_COLOUR_PRESET),
     SET_FIELD(ColourPreview, { static_cast<uint32_t>(SPR_NONE), static_cast<uint32_t>(SPR_NONE) }),
-    SET_FIELD(ColourKey, RideColourKey::Ride)
+    SET_FIELD(ColourKey, RideColourKey::Ride),
+    SET_FIELD(TrackBehaviours,{TrackElemTypeAlternateName::Default,TrackElemTypeAlternateName::LogBumps})
```

Comment:
all of the ridedata headers do not have clang format enabled so you need to space them out yourself. Could you go through and make it so that it follows the rest of the codebase. I know its a pain doing that but its also a pain looking at it.
e.g.:
```
SET_FIELD(TrackBehaviours, { TrackElemTypeAlternateName::Default, TrackElemTypeAlternateName::LogBumps }),
```

Reply:
This default is used for a lot of RTD's. Its a bit of an odd default as it refers to LogBumps i think it would be better if we had a:
```
constexpr RideTrackBehaviours _defaultTrackBehaviour = { TrackElemTypeAlternateName::Default, TrackElemTypeAlternateName::LogBumps };
```
in this header and refer to that for all of the RTD's that use the default. That makes it all easy to understand why we are talking about logbumps.

Reply:
I chose Rapids as the default value for that field because that's what the TrackElemType is named. My thinking with the defaults was that any powered ride got the SpeedController and any water ride got the Rapids default.